docs/get-started/getting-started-on-prem.md

```diff
@@ -302,8 +302,9 @@ to setup a Kubernetes cluster.
   [GPU discovery script](#install-the-gpu-discovery-script) on the node from which you are
```

Comment:
perhaps we should be very explicity in the first paragraph to see the Dockerfile that shows putting the spark rapids jars into the image to address https://github.com/NVIDIA/spark-rapids/issues/664

Reply:
Please review updated comments.